QUARKUS-874.md

```diff
@@ -23,26 +30,25 @@ NOTE: We will prepare steps to get Windows 10 instance to be able to handle situ
    - New Jenkins Jobs
  - Combinations of Quarkus extensions Test Suite
    - In Github CI - for Pull Requests and Daily runs
- - OpenShift Test Suite
-   - New Jenkins Jobs on selected modules only (the ones using deployment strategies)
-
-### Platform testing
-JVM mode on is the focus for Windows platform. This RFE does not include native executable (Mandrel) or any extension that requires native libraries (like Kafka streams client).
+ - Quarkus QE Test Suite
+   - In Github CI - for Pull Requests and Daily runs
+   - New weekly Jenkins Job running against Quarkus main
 
 ## Impact on Jenkins
  - New Jobs definitions
  - Rename existing jobs to differentiate them for those running on RHEL and on Windows platforms
+ - New Jenkins AWS runners as our tests rely mostly on the TestContainers, we need Linux containers and hardware virtualization respectively, which is not possible in the RHOS-D.
 
 ## Impact on resources
 The major impact is on the PSI and Jenkins capacity:
- - Windows instances will be executed next to the RHEL instances 
-   - Overall test execution for RHEL and Windows jobs will be affected (doubled in the worse case)
-   - Windows virtual machines usually require more resources than RHEL ones, expecting large or xlarge flavor to be used
-   - More CPU / memory in the lab would be needed to ensure similar execution time as now.
- - Remote Docker machines instances based on RHEL to run the containers with services like DBs / Keycloak / Kafka / Infinispan for Windows jobs
-   - Windows on PSI does not support Linux containers yet, so we want to try to use RHEL instances to run the containers as a workaround
+ - Windows instances will be created from RHEL instances, therefore we need at least one RHEL instance of medium flavor for each job.
+ - Windows instances that requires Docker will be run in AWS, therefore only impact to the current capacity is above-mentioned RHEL instance. 
+ - Windows instances that do not require Docker will be executed next to the RHEL instances, but they are mostly lightweight short-running jobs
+ - Overall test execution for RHEL and Windows jobs will be affected (doubled in the worse case)
  - More jobs pulling artifacts from Artifactory instance
-   - We may need to use more resources for Artifactory
+ - We may need to use more resources for Artifactory
+ - Weekly Test Suite run on AWS will require around 9.88 h per week
```

Comment:
This number will grow as the TS size grows, add a note about it.

Reply:
Appreciated!